src/browser/data/equipment.json

```diff
@@ -0,0 +1,75 @@
+{
+  "desc" : "There are some tools we equip you with, and opportunities we want to give you, to make your way more exiciting.",
+  "options" : [
+    {
+      "id" : 0,
+      "title" : "Tools",
+      "img" : "assets/img/equipment/tools.svg",
+      "active" : "assets/img/equipment/tools-active.svg"
+    },
+    {
+      "id" : 1,
+      "title" : "Opportunities",
+      "img" : "assets/img/equipment/opportunities.svg",
+      "active" : "assets/img/equipment/opportunities-active.svg"
+    },
+    {
+      "id" : 2,
+      "title" : "Personal Development",
+      "img" : "assets/img/equipment/pdevelop.svg",
+      "active" : "assets/img/equipment/pdevelop-active.svg"
+    }
+  ],
+  "0": [
+    "Best work equipment (we are Apple crazy but you always have a choice if you are not)",
```

Comment:
Not sure how to comment on this, but there's a design problem with this screen - there's no clean indication that `CHECK YOUR EQUIPMENT` screen as 3 tabs. I went through it three times and didn't notice it

Reply:
In general, there are 12 slides (10 steps + 1 has 3 tabs). That's a lot considering that first 2 bring nothing to the table and the rest is packed with "fluff" that might not be that interesting at this point (for instance: `Regular employee evaluation` or `Meeting your buddy/mentor` IMO aren't useful before joining)

Reply:
I've changed color of active tab title. I think it's much more intuitive for the user right now. When it comes about your points, I'll send it to talent team.